server/package.json

```diff
@@ -13,15 +13,18 @@
     "dotenv": "^7.0.0",
     "express": "^4.16.4",
     "helmet": "^3.16.0",
-    "mongoose": "^5.5.4"
+    "mongoose": "^5.5.4",
+    "socket.io": "^2.2.0",
+    "chess.js": "https://github.com/jhlywa/chess.js#master"
```

Comment:
You may not like this one. But ideally we keep these modules in alphabetic order.
It's just a preference. There are vscode plugins which do it automatically for me. But you can skip this.

Reply:
Done.